### ISSUE TITLE
Can't add to PATH environment table on non ASCII username in Windows

### ISSUE THREAD

**Issue by Adamillo** [nim-lang/Nim#20224]
### What happened?

Nim cannot be added to the PATH environment table to a non ASCII path.

### Nim Version

Nim 1.6.6

### Expected Standard Output Logs

```shell
Nim should be able to add to the PATH environmental table in any path, regardless if it's ASCII or not
```

### Possible Solution

To be able to automatically install Nim to the PATH environment table Nim should have UTF-8 support

**Comment by havardjohn** [comment]
I have the same issue, and posted it as issue #20083. It should be fixed with this PR: #20084

**Comment by Adamillo** [comment]
I see, so this is a clone of the same issue. I guess I should close it then
